examples/all-clusters-app/linux/include/bridged-actions-stub.cpp

```diff
@@ -0,0 +1,110 @@
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/****************************************************************************
+ * @file
+ * @brief Stub implementation for the Bridged Actions Cluster
+ ***************************************************************************/
+
+#include <app-common/zap-generated/af-structs.h>
+#include <app-common/zap-generated/cluster-objects.h>
+#include <app-common/zap-generated/ids/Attributes.h>
+#include <app-common/zap-generated/ids/Clusters.h>
+#include <app/AttributeAccessInterface.h>
+#include <app/util/attribute-storage.h>
+#include <lib/support/CodeUtils.h>
+#include <lib/support/logging/CHIPLogging.h>
+
+using namespace chip;
+using namespace chip::app;
+using namespace chip::app::Clusters;
+using namespace chip::app::Clusters::BridgedActions::Attributes;
+
+namespace {
+
+class BridgedActionsAttrAccess : public AttributeAccessInterface
+{
+public:
+    // Register for the Bridged Actions cluster on all endpoints.
+    BridgedActionsAttrAccess() : AttributeAccessInterface(Optional<EndpointId>::Missing(), BridgedActions::Id) {}
+
+    CHIP_ERROR Read(const ConcreteAttributePath & aPath, AttributeValueEncoder & aEncoder) override;
+
+private:
+    static constexpr uint16_t ClusterRevision = 1;
+
+    CHIP_ERROR ReadActionListAttribute(EndpointId endpoint, AttributeValueEncoder & aEncoder);
+    CHIP_ERROR ReadEndpointListAttribute(EndpointId endpoint, AttributeValueEncoder & aEncoder);
+    CHIP_ERROR ReadSetupUrlAttribute(EndpointId endpoint, AttributeValueEncoder & aEncoder);
+    CHIP_ERROR ReadClusterRevision(EndpointId endpoint, AttributeValueEncoder & aEncoder);
+};
+
+CHIP_ERROR BridgedActionsAttrAccess::ReadActionListAttribute(EndpointId endpoint, AttributeValueEncoder & aEncoder)
+{
+    // Just return an empty list
+    return aEncoder.EncodeList([](const TagBoundEncoder & encoder) -> CHIP_ERROR { return CHIP_NO_ERROR; });
+}
+
+CHIP_ERROR BridgedActionsAttrAccess::ReadEndpointListAttribute(EndpointId endpoint, AttributeValueEncoder & aEncoder)
+{
+    // Just return an empty list
+    return aEncoder.EncodeList([](const TagBoundEncoder & encoder) -> CHIP_ERROR { return CHIP_NO_ERROR; });
+}
+
+CHIP_ERROR BridgedActionsAttrAccess::ReadSetupUrlAttribute(EndpointId endpoint, AttributeValueEncoder & aEncoder)
+{
+    const char SetupUrl[] = "www.signify.com";
+    return aEncoder.Encode(chip::Span<const char>(SetupUrl, sizeof(SetupUrl)));
```

Comment:
```suggestion
    return aEncoder.Encode(chip::Span<const char>(SetupUrl, strlen(SetupUrl)));
```
Otherwise you are sticking the null-terminator in there too, which is not right.

Reply:
Accepted and implemented.